client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -89,24 +91,30 @@ const MarketplaceThankYou = ( { productSlug }: { productSlug: string } ) => {
 		! new URLSearchParams( document.location.search ).has( 'hide-progress-bar' )
 	);
 
-	const areAllPluginsOnSite =
-		!! pluginsOnSite.length && pluginsOnSite.every( ( pluginOnSite ) => !! pluginOnSite );
+	// Retrieve theme information
+	const wpComThemes = useSelector( ( state ) => getThemes( state, 'wpcom', productSlugs ) );
+
+	const isAllDataFetched =
+		!! pluginsOnSite.length &&
+		pluginsOnSite.every( ( pluginOnSite, index ) => !! pluginOnSite || !! wpComThemes[ index ] );
 
 	// Consolidate the plugin information from the .org and .com sources in a single list
-	const pluginInformationList = useMemo( () => {
+	const productInformationList = useMemo( () => {
 		return pluginsOnSite.reduce(
 			( pluginsList: Array< any >, pluginOnSite: Plugin, index: number ) => {
 				pluginsList.push( {
 					...wpComPluginsData[ index ],
 					...wporgPlugins[ index ],
 					...pluginOnSite,
+					...wpComThemes[ index ],
+					isTheme: !! wpComThemes[ index ],
```

Comment:
instead of `isTheme` boolean, would it be more useful having a `ProductType` enumeration?

Reply:
Updated here https://github.com/Automattic/wp-calypso/pull/74125/commits/c69433cc985af19d544643499090efbcd55c7d80